infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/LogEntrySender.java

```diff
@@ -132,6 +138,11 @@ public void send(UUID logEntrySyncEventId) {
                 cancelLogEntrySync(LogReplicationError.LOG_ENTRY_MESSAGE_SIZE_EXCEEDED,
                         LogReplicationEventType.SYNC_CANCEL, logEntrySyncEventId);
                 return;
+            } catch (GroupDestinationChangeException gce) {
+                cancelLogEntrySync(LogReplicationError.GROUP_DESTINATION_CHANGE,
+                        LogReplicationEventType.SYNC_CANCEL, logEntrySyncEventId);
+                SnapshotSyncUtils.enforceSnapshotSync(logReplicationFSM.getSession(), replicationContext.getCorfuStore());
```

Comment:
explicit trigger of a forced snapshot sync is not required.  The SYNC_CANCEL event causes a state transition to snapshot sync.

Reply:
I see, didn't realize that. Removed the manual trigger and the getter from LogReplicationContext. Thanks!